db/sql/05_msar.sql

```diff
@@ -4320,11 +4320,21 @@ SELECT val::text;
 $$ LANGUAGE SQL IMMUTABLE RETURNS NULL ON NULL INPUT PARALLEL SAFE;
 
 
+CREATE OR REPLACE FUNCTION msar.format_data(val jsonb[]) returns jsonb AS $$
+SELECT jsonb_agg(x::text) FROM unnest(val) AS x;
```

Comment:
Why not aggregate to an array? That would be more consistent with the data explorer experience. So, the output would be `text[]`.